libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -401,12 +402,29 @@ public MapboxMap getMapboxMap() {
     return map;
   }
 
+  public void setInstructionViewCallback() {
```

Comment:
This method can be `private` and we can be more specific in the naming `initializeInstructionListListener`.  I think aligning `callback` and `listener` here makes sense even though they are essentially the same.